### PR TITLE
docs: review multimodal search

### PR DESCRIPTION
**Pull request by bwanglzu**
Review multi & cross modal search document.

1. Rewrite the document in `rst` based on our template.
2. Some small updates on wording and sections.

Checkout the example lively [here](https://deploy-preview-61--jina-docs.netlify.app/chapters/cross-multi-modality/index.html).

### CODE REVIEW

File: chapters/cross-multi-modality/index.rst [hunk 1]
```diff
@@ -8,6 +8,49 @@ Multi-modal and Cross-modal Search in Jina
...
+
+For this reason, and to have first-class support for cross and multi-modal search,
+Jina offers modality as an attribute in its Document primitive type.
+Now that we agreed on the concept of modality,
+we can describe cross-modal and multi-modal search.
+
+ - Cross-modal search can be defined as a set of retrieval applications that try to effectively find relevant documents of modality A by querying with documents from modality B.
+ - Multi-modal search can be defined as a set of retrieval applications that try to effectively project documents of different modalities into a common embedding space, and find relevant documents with respect to the fusion of multiple modalities
+
+The main difference between these two search modes is that for cross-modal, there is a direct mapping between a single document or chunk and a
```
Comment (JoanFM) [inline chapters/cross-multi-modality/index.rst]: avoid the usage of `chunk`

**Review by CatStark — APPROVED** [review]
Great doc!

File: chapters/cross-multi-modality/index.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+For this reason, and to have first-class support for cross and multi-modal search,
+Jina offers modality as an attribute in its Document primitive type.
+Now that we agreed on the concept of modality,
+we can describe cross-modal and multi-modal search.
+
+ - Cross-modal search can be defined as a set of retrieval applications that try to effectively find relevant documents of modality A by querying with documents from modality B.
+ - Multi-modal search can be defined as a set of retrieval applications that try to effectively project documents of different modalities into a common embedding space, and find relevant documents with respect to the fusion of multiple modalities
+
+The main difference between these two search modes is that for cross-modal, there is a direct mapping between a single document and a
+vector in embedding space, while for MultiModal this does not hold true, since 2 or more documents might be combined into a single vector.
```
Comment (FionnD) [inline chapters/cross-multi-modality/index.rst]: MultiModal -> Make sure you always use the hyphen

File: chapters/cross-multi-modality/index.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Supporting cross-modal search in Jina is very easy.
+User just needs to properly set the modality field of the input documents and design the Flow in such a way that the queries target the desired embedding space.
+
+We have created an example project follow the cross-modal search manner.
+The `Image Search using Captions <https://github.com/jina-ai/examples/tree/master/cross-modal-search>`_ example allows user search for images by giving corresponding caption descriptions.
+
+We encode images and its captions (any descriptive text of the image) in separate indexes,
+which are later queried in a cross-modal fashion.
+It queries the text index using image embeddings and query the image index using text embeddings.
```
Comment (FionnD) [inline chapters/cross-multi-modality/index.rst]: queries

File: chapters/cross-multi-modality/index.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+The main difference between these two search modes is that for cross-modal, there is a direct mapping between a single document and a
+vector in embedding space, while for MultiModal this does not hold true, since 2 or more documents might be combined into a single vector.
+
+This unlocks a lot of powerful patterns and makes Jina fully flexible and agnostic to what can be searched.
+
+Cross modal search
+--------------------
+
+Supporting cross-modal search in Jina is very easy.
+User just needs to properly set the modality field of the input documents and design the Flow in such a way that the queries target the desired embedding space.
```
Comment (FionnD) [inline chapters/cross-multi-modality/index.rst]: **A** user

File: chapters/cross-multi-modality/index.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+We have created an example project follow the cross-modal search manner.
```
Comment (FionnD) [inline chapters/cross-multi-modality/index.rst]: **that follows**

File: chapters/cross-multi-modality/index.rst [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+The `Image Search using Captions <https://github.com/jina-ai/examples/tree/master/cross-modal-search>`_ example allows user search for images by giving corresponding caption descriptions.
```
Comment (FionnD) [inline chapters/cross-multi-modality/index.rst]: allows users to search

**Review by FionnD — CHANGES_REQUESTED** [review]
A very good document, just some small grammar issues. 

No need for me to review again!

File: chapters/cross-multi-modality/index.rst (hunk 4 above)
Comment (CatStark) [reply chapters/cross-multi-modality/index.rst]: I think "user" is correct or? But if we want to specify an user, it should be "A**n** user"

**Comment by JoanFM** [comment]
Lets remove the wording new, as this is supposed to last long and these are just the constructs we offer

**Comment by bwanglzu** [comment]
@JoanFM done

**Review by JoanFM — APPROVED** [review]

**Pull request merged**